orttraining/orttraining/test/gradient/gradient_ops_test.cc

```diff
@@ -2423,6 +2423,24 @@ TEST(GradientCheckerTest, ClipGrad) {
   }
 }
 
+TEST(GradientCheckerTest, TileGrad) {
+  float max_error;
+  GradientChecker<float, float, float> gradient_checker;
+  OpDef op_def{"Tile", kOnnxDomain, 10};
+
+  // default values for optional tensors like axes and steps.
+  {
+    TensorInfo x_info({2, 4}, true);
+    TensorInfo repeat_info({2,2}, false);
```

Comment:
One more test case for 1D and 3D input, 
one test case for repeating of {1s}

Reply:
done